Tests/Tests/RouterSpec.swift

```diff
@@ -450,6 +593,34 @@ class RouterSpec: QuickSpec {
                         expect(displayedTooltip.frame.origin).toEventually(equal(lastTooltipPosition.applying(translation)))
                     }
 
+                    it("will show tooltip's in different position") {
+                        var tooltipLastPosition = CGPoint(x: 0, y: 0)
+                        let window = UIApplication.shared.getKeyWindow()!
+                        for position in [
+                            TooltipBodyData.Position.topLeft,
+                            TooltipBodyData.Position.topRight,
+                            TooltipBodyData.Position.topCenter,
+                            TooltipBodyData.Position.bottomLeft,
+                            TooltipBodyData.Position.bottomRight,
+                            TooltipBodyData.Position.bottomCenter,
+                            TooltipBodyData.Position.left,
+                            TooltipBodyData.Position.right
+                        ] {
+                            let tooltip = TestHelpers.generateTooltip(id: "test", position: position)
+                            router.displayTooltip(tooltip,
+                                                  targetView: targetView,
+                                                  identifier: TooltipViewIdentifierMock,
+                                                  imageBlob: imageBlob,
+                                                  becameVisibleHandler: { _ in },
+                                                  confirmation: true,
+                                                  completion: { _ in })
+                            expect(window.findTooltipView()).toEventuallyNot(beNil())
+                            let displayedTooltip = window.findTooltipView()!
```

Comment:
It's a coincidence that `findTooltipView()` returns the most recently displayed tooltip. Before calling this function you should remove the displayed tooltip from the previous loop to ensure accurate and stable test result.

Reply:
Update here [3cee1ec](https://github.com/rakutentech/ios-inappmessaging/pull/275/commits/3cee1ec04fdbedfab437064f59c968abeb121232).